alembic/operations/ops.py

```diff
@@ -959,6 +959,7 @@ def __init__(
         self.comment = kw.pop("comment", None)
         self.prefixes = kw.pop("prefixes", None)
         self.kw = kw
+        self.table_info = kw.pop('info', {})
```

Comment:
**mike bayer** (zzzeek) wrote:

call this ".info"

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/alembic/+/2989